bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -441,6 +472,9 @@ spec:
       kind: NonAdminBackup
       name: nonadminbackups.oadp.openshift.io
       version: v1alpha1
+    - kind: NonAdminBackupStorageLocation
```

Comment:
you have to manually add entry in `config/manifests/bases/oadp-operator.clusterserviceversion.yaml` to `description` and `displayName` to be present 

example https://github.com/openshift/oadp-operator/pull/1599/files

Reply:
Done, also added one small change to cover the NonAdminBackupStorageLocation within Phases that are used across NonAdminBackup, NonAdminRestore and the NonAdminBackupStorageLocation.